src/main/java/org/jitsi/videobridge/IceDtlsTransportManager.java

```diff
@@ -136,9 +134,14 @@ public void setEndpoint(Endpoint endpoint)
     @Override
     public boolean isConnected()
     {
-        return iceAgent.getState().isEstablished();
+        // TODO: do we consider this TM connected when ICE completes, or when
+        // ICE and DTLS both complete?
```

Comment:
good question, i'd have to check what all relies on this.  does seem like there's a case to be made for it waiting for DTLS?

Reply:
It looks like the only thing that waits for ICE, but not DTLS is IceDtlsTM (I even removed the listeners for "ice connected" in one of the commits). So I think we should refactor and simplify further at some point.